backend/src/Constant/StatusConstant.php

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Status for what?
filename must be more clarify

Reply:
ok